### ISSUE TITLE
What graphical card are you using for video prediction and corresponding training time

### ISSUE THREAD

**Issue by umichyiwan** [tensorflow/models#537]
## video prediction
Hi @cbfinn , I would like to ask what graphical card are you using in your experiment and what is your training time.

**Comment by cbfinn** [comment]
On a standard NVIDIA Titan X GPU, full training (100k iterations) should take a few days, depending on the model options.

That said, you should see reasonable predictions after only 15k iterations.

Let me know if you experience much slower training than expected.

**Comment by umichyiwan** [comment]
100k iterations? You mean iterate 50000 training video clips for 100k times?

**Comment by cbfinn** [comment]
By 100k iterations, I mean 100k gradient steps, not 100k epochs.

**Comment by umichyiwan** [comment]
Thank you!

**Comment by jart** [comment]
Yes we love our Titan X cards. There are empty boxes of them strewn all over the place in the Google Brain building.

**Comment by falcondai** [comment]
@umichyiwan took ~41 hrs on my titan X

**Comment by umichyiwan** [comment]
Thanks for you guys.
I noticed that your video prediction model does well in terms of end-effector. However, object motion it predicts is not satisfiable. Objects being moved in GT just become blurring in the prediction result. In most of the situations, they actually don't move. 
I guess the reason end-effector performs well is because you directly input current and target gripper poses to the network. And this is highly related to pixel flow.
But why objects perform not well? What do you guys think of it.

**Comment by cbfinn** [comment]
See https://github.com/tensorflow/models/issues/623
